src/components/Select/Select.tsx

```diff
@@ -165,8 +165,18 @@ const Select = React.forwardRef<HTMLInputElement, SelectProps>((props, ref) => {
       }
     } else {
       if (onChange) {
-        // @ts-ignore
-        onChange(option);
+        if (option.isCreated) {
+          const newOption: SelectOption = {
+            value: option.value,
+            label: option.value.toString(),
+            isCreated: true,
+          };
+          // @ts-ignore
```

Comment:
why we are ingoring here?

Reply:
@kostasdano reminder

Reply:
its because of the conditional types used for onChange:
```
(single-select)
onChange?: (selectedOption?: SelectOption) => void;

(multi-select)
onChange?: (selectedOption: SelectOption[]) => void;
```

we had some issues with this approach so we left some ts-ignore comments needed in order for this implementation to work